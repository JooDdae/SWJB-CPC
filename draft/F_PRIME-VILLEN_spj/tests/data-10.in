463539391917899497867720994396476427
