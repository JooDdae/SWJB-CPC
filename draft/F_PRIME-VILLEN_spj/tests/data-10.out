12
90 711 157 0 4 24 1177 1080 317 85 118 836 